### PR TITLE
Access updates

### PR DESCRIPTION
**Pull request by gknisely**
updated uses for country admin logic.

### CODE REVIEW

File: valhalla/baldr/graphconstants.h [hunk 1]
```diff
@@ -205,7 +205,10 @@ enum class Use : uint8_t {
 
   // Pedestrian specific uses
   kFootway = 25,
-  kSteps   = 26,           // Stairs
+  kPedestrian = 26,
+  kPath = 27,
+  kSteps = 28,             // Stairs
```
Comment (kevinkreiser) [inline valhalla/baldr/graphconstants.h]: any reason for changing kSteps to 28? you could have kPedestrian be 28 and keep steps the same avoid confusion? maybe it doesnt matter at all though :smile:

Comment (gknisely) [inline valhalla/baldr/graphconstants.h]: yes to keep the foot paths together....that is all

Comment (kevinkreiser) [inline valhalla/baldr/graphconstants.h]: looks like it doesnt really matter though in the end, just used for a switch case in mjolnir so all good

**Comment by kevinkreiser** [comment]
:ship:

File: valhalla/baldr/graphconstants.h (hunk 1 above)
Comment (dnesbitt61) [inline valhalla/baldr/graphconstants.h]: Do not change values of existing uses if you can help it. That would mean that server and data producer would have to be deployed together (steps is used in costing and costing filters)

Comment (dnesbitt61) [inline valhalla/baldr/graphconstants.h]: Also - are pedestrian and path basically footway after all logic is complete? Or would this require several additional if statements in costing to determine if the edge should be favored for pedestrian routes?

**Comment by gknisely** [comment]
updated

**Pull request merged**